src/Quartz.HttpClient/HttpClient/HttpScheduler.cs

```diff
@@ -0,0 +1,476 @@
+﻿using System.Text.Json;
+
+using Quartz.HttpApiContract;
+using Quartz.Impl.Matchers;
+using Quartz.Spi;
+
+namespace Quartz.HttpClient;
+
+public class HttpScheduler : IScheduler
```

Comment:
does this need to be public?

Reply:
At the moment we do not have any factory method to create this, so need to be public for consumers to be able to create new instance.